src/App.tsx

```diff
@@ -364,4 +371,21 @@ const App: React.FC = () => {
 
 export default App;
 
+const LightBGColor = styled.div`
```

Comment:
I think you can delete LightBGColor and DarkBGColor  from here... I added them inside another component already